### PR TITLE
USHIFT-4350: Download cf-gen.yaml from MicroShift repository

### PR DESCRIPTION
**Pull request by vanhalenar**
After merging this [PR](https://github.com/openshift/microshift/pull/4018), we will have the `cf-gen.yaml` file in the MicroShift repo. We can `curl` the file from there instead of having two versions of it, while also making the script cleaner.

### CODE REVIEW

File: ci-operator/step-registry/openshift/microshift/infra/aws/ec2/openshift-microshift-infra-aws-ec2-commands.sh [hunk 1]
```diff
@@ -6,6 +6,9 @@ source "${SHARED_DIR}/ci-functions.sh"
 trap_subprocesses_on_term
 trap_install_status_exit_code $EXIT_CODE_AWS_EC2_FAILURE
 
+cf_gen=$(mktemp /tmp/cf_gen.XXXXXXXX.yaml)
+curl -o "${cf_gen}" https://raw.githubusercontent.com/openshift/microshift/refs/heads/aws/scripts/aws/cf-gen.yaml
```
Comment (ggiguash) [inline ci-operator/step-registry/openshift/microshift/infra/aws/ec2/openshift-microshift-infra-aws-ec2-commands.sh:10]:
```suggestion
curl -o "${cf_gen}" https://raw.githubusercontent.com/openshift/microshift/refs/heads/main/scripts/aws/cf-gen.yaml
```

File: ci-operator/step-registry/openshift/microshift/infra/aws/ec2/openshift-microshift-infra-aws-ec2-commands.sh [hunk 2]
```diff
@@ -71,329 +74,7 @@ function save_stack_events_to_shared()
...
-  PrivateIp:
-    Description: The bastion host Private DNS, will be used for cluster install pulling release image
-    Value: !GetAtt RHELInstance.PrivateIp
-  PublicIp:
-    Description: The bastion host Public IP, will be used for registering minIO server DNS
-    Value: !GetAtt RHELInstance.PublicIp
-EOF
-
-
+cat "${cf_gen}" > "${cf_tpl_file}"
```
Comment (ggiguash) [inline ci-operator/step-registry/openshift/microshift/infra/aws/ec2/openshift-microshift-infra-aws-ec2-commands.sh:77]: I'm not sure we need another temporary file for download. Can we download directly to ${cf_tpl_file}?

File: ci-operator/step-registry/openshift/microshift/infra/aws/ec2/openshift-microshift-infra-aws-ec2-commands.sh (hunk 1 above)
Comment (vanhalenar) [reply ci-operator/step-registry/openshift/microshift/infra/aws/ec2/openshift-microshift-infra-aws-ec2-commands.sh]: fixed

File: ci-operator/step-registry/openshift/microshift/infra/aws/ec2/openshift-microshift-infra-aws-ec2-commands.sh (hunk 2 above)
Comment (vanhalenar) [reply ci-operator/step-registry/openshift/microshift/infra/aws/ec2/openshift-microshift-infra-aws-ec2-commands.sh]: changed

**Pull request merged**